cypress/e2e/read-receipts/high-level.spec.ts

```diff
@@ -801,32 +877,100 @@ describe("Read receipts", () => {
     });
 
     describe("reactions", () => {
-        // Justification for this section: edits an reactions are similar, so we
-        // might choose to miss this section, but I have included it because
-        // edits replace the content of the original event in our code and
-        // reactions don't, so it seems possible that bugs could creep in that
-        // affect only one or the other.
-
         describe("in the main timeline", () => {
-            it.skip("Reacting to a message makes a room unread", () => {});
-            it.skip("Reading a reaction makes the room read", () => {});
-            it.skip("Marking a room as read after a reaction makes it read", () => {});
-            it.skip("Reacting to a message after marking as read makes the room unread", () => {});
-            it.skip("A room with a reaction is still unread after restart", () => {});
-            it.skip("A room where all reactions are read is still read after restart", () => {});
+            it("Receiving a reaction to a message does not make a room unread", () => {
+                goTo(room1);
+                assertRead(room2);
+                receiveMessages(room2, ["Msg1", "Msg2"]);
+                assertUnread(room2, 2);
+
+                // When I read the main timeline
+                goTo(room2);
+                assertRead(room2);
+
+                goTo(room1);
+                receiveMessages(room2, [reactionTo("Msg2", "🪿")]);
+                assertRead(room2);
```

Comment:
Do we need to wait for a while here? Otherwise we might just read state from before the reaction arrives

Reply:
(same question for some others below.)

Reply:
> Do we need to wait for a while here?

Possibly, maybe receiveMessages needs to somehow await their receipt & effect all over the shop